deploy/olm-catalog/argocd-operator/0.10.0/argoproj.io_argocds.yaml

```diff
@@ -49,6 +49,10 @@ spec:
           spec:
             description: ArgoCDSpec defines the desired state of ArgoCD
             properties:
+              aggregatedClusterRoles:
```

Comment:
I think we can just remove the contents of this file entirely from the PR, unless you are convinced it is beneficial. I removed, the file, ran `make bundle`, and it was not generated, which to me says it is not needed (or should not be included). WDYT?

Reply:
Yes @jgwest it was generated because of old version, it have removed it. PTAL.